src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -2548,6 +2592,41 @@ private IEnumerable<string> IterateCollectionNames(Transaction tx, JsonOperation
             return result;
         }
 
+        public IEnumerable<Document> GetDocumentsByBucketFrom(DocumentsOperationContext context, int bucket, long etag)
+        {
+            var table = new Table(DocsSchema, context.Transaction.InnerTransaction);
```

Comment:
shouldn't this be `OpenTable()` ?

Reply:
I don't think we need to use `OpenTable()` here

`table =  new Table(schema, tx)` is meant to be used for global reads only, when we want to use a global index to find data without touching the actual table - which is what we're doing here.

AFAIK `OpenTable(schema, name)` is used for writes, and for reading collection-specific data, e.g. :
 `var table = context.Transaction.InnerTransaction.OpenTable(DocsSchema, "Collection.Documents.users"))`

Reply:
`OpenTable` implements a cache, so can avoid allocations. The global reads is when we don't have enough information, but here we should have that, no?